CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+- #661 - Panic when image does not exist and running on CI or with `CROSS_FORCE_IMAGE=1`
```

Comment:
Should it accept a certain range of values (since setting it to `false` or `0` should be falsey, which `true` or `1` should be truthy) rather than merely check the existence of the variable? This looks like it could also be relevant to `CROSS_DOCKER_IN_DOCKER`.